README.md

```diff
@@ -23,6 +23,30 @@ npm run build:prod
 npm start:prod
 ```
 
+### CRUD API
+|    Method    |           Endpoint          |                   Action                |
+| ------------ | --------------------------- | --------------------------------------- |
+| **POST**     |  /api/allreviews/:id        |  CREATE a new review for an item id     |
+| **GET**      |  /api/allreviews/:id        |  READ and return reviews for an item id |
+| **GET**      |  /api/allreviews/review:id  |  READ and return a review by review id  |
+| **PUT**      |  /api/allreviews/review/:id |  UPDATE review by review id             |
+| **DELETE**   |  /api/allreviews/review/:id |  DELETE review by review id             |
+
+**Example Response for /api/allreviews/review/:id**
+```sh
+{
+  "itemId": 1,
+  "name": "Reviewing Hobbyist",
+  "stars": "4.5",
+  "date": "January 10, 2020",
+  "review": "Gone are the days of biting off slice-sized chunks of banana and spitting them onto a serving tray…. Next on my wish list: a kitchen tool for dividing frozen water into cube-sized chunks.",
+  "image": "",
+  "title": "Best Hutzler 571 Banana Slicer",
+  "avatar": 2,
+  "foundThisHelpful": 1952
+}

```

Comment:
this is a great idea. will be adding to mine, additionally might be good to give instructions on what the update route is looking for here as well.

Reply:
The example was going to be the same so I updated the description instead. Thanks for the idea!